src/schemas/execute.yaml

```diff
@@ -0,0 +1,435 @@
+    state:
+      title: Storage
+      description: Key-value mapping to override all slots in the account storage before executing the call. This functions similar to eth_call's state parameter.
+      $ref: '#/components/schemas/AccountStorage'
+AccountOverrideStateDiff:
+  title: Account override with partial storage modification
+  required:
+    - stateDiff
+  properties:
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    movePrecompileToAddress:
+      title: MovePrecompileToAddress
+      $ref: '#/components/schemas/address'
+      description: Moves addresses precompile into the specified address. This move is done before the 'code' override is set. Can only move precompiles.
+    stateDiff:
+      title: Storage difference
+      description: Key-value mapping to override individual slots in the account storage before executing the call. This functions similar to eth_call's state parameter.
+      $ref: '#/components/schemas/AccountStorage'
+AccountStorage:
+  title: Storage slots for an account
+  type: object
+  patternProperties:
+    '^0x[a-fA-F0-9]{64}$':
+      $ref: '#/components/schemas/hash32'
+  additionalProperties: false
+BlockOverrides:
+  title: Context fields related to the block being executed
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint64'
+      description: When overriding block numbers across multiple blocks, block number need to be increasing. Skipping over blocks numbers is possible. If block number is not specified, it's incremented by one for each block.
+    prevRandao:
+      title: The Previous value of randomness beacon
+      $ref: '#/components/schemas/uint256'
+    time:
+      title: Time
+      $ref: '#/components/schemas/uint64'
+      description: When overriding Time across multiple blocks, Time need to be increasing. If time is not specified, it's incremented by one for each block.
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
+      title: Fee Recipient (also known as coinbase)
+      $ref: '#/components/schemas/address'
+    baseFeePerGas:
+      title: Base fee per unit of gas
+      $ref: '#/components/schemas/uint256'
+    withdrawals:
+      title: Withdrawals made by validators
+      $ref: '#/components/schemas/Withdrawals'
+    blobBaseFee:
+      title: Base fee per unit of blob gas
+      $ref: '#/components/schemas/uint256'
+Withdrawals:
+  title: Withdrawals made by validators
+  description: This array can have a maximum length of 16.
+  type: array
+  items:
+    - $ref: '#/components/schemas/Withdrawal'
+Withdrawal:
+  title: A withdrawal made by a validator
+  type: object
+  properties:
+    index:
+      title: index
+      $ref: '#/components/schemas/uint64'
+    validator_index:
```

Comment:
Can we go with a style that is consistent with all other fields like ValidatorIndex (no "_")?